src/Form/Field.php

```diff
@@ -213,47 +213,43 @@ public static function defaults(): array
             'computed' => [
                 'after' => function () {
                     /** @var \Kirby\Form\Field $this */
-                    if ($this->after !== null) {
-                        return $this->model()->toString($this->after);
-                    }
+                    return $this->toString($this->after);
                 },
                 'before' => function () {
                     /** @var \Kirby\Form\Field $this */
-                    if ($this->before !== null) {
-                        return $this->model()->toString($this->before);
-                    }
+                    return $this->toString($this->before);
                 },
                 'default' => function () {
                     /** @var \Kirby\Form\Field $this */
-                    if ($this->default === null) {
-                        return;
-                    }
-
-                    if (is_string($this->default) === false) {
-                        return $this->default;
-                    }
-
-                    return $this->model()->toString($this->default);
+                    return $this->toString($this->default);
                 },
                 'help' => function () {
                     /** @var \Kirby\Form\Field $this */
-                    if ($this->help) {
-                        $help = $this->model()->toString($this->help);
-                        $help = $this->kirby()->kirbytext($help);
-                        return $help;
+                    if ($text = $this->toString($this->help)) {
+                        return $this->kirby()->kirbytext($text);
                     }
                 },
                 'label' => function () {
                     /** @var \Kirby\Form\Field $this */
-                    if ($this->label !== null) {
-                        return $this->model()->toString($this->label);
-                    }
+                    return $this->toString($this->label);
                 },
                 'placeholder' => function () {
                     /** @var \Kirby\Form\Field $this */
-                    if ($this->placeholder !== null) {
-                        return $this->model()->toString($this->placeholder);
+                    return $this->toString($this->placeholder);
+                }
+            ],
+            'methods' => [
+                'toString' => function ($value = null) {
+                    if ($value === null) {
+                        return;
                     }
+
+                    if (is_string($value) === false) {
+                        return $value;
+                    }
```

Comment:
@lukasbestle I think we have an issue here too. For non-string values, right? Will it solve if we remove the return type of method?

Reply:
Pinging @bastianallgeier @distantnative to review

Reply:
I thinking solving this in the fields themselves would require us to define the `default` prop in all these fields explicitly, while @afbora tried to rather use a generally defined one from `Kirby\Form\Field`. To me both would be fine, handling this in the fields, but I also wouldn't mind if `toString()` includes a check... could be a bit DRYer.